admin/code/LeftAndMain.php

```diff
@@ -339,8 +339,6 @@ public function init() {
 
 		if (Director::isDev()) Requirements::javascript(FRAMEWORK_ADMIN_DIR . '/javascript/leaktools.js');
 
-		HTMLEditorField::include_js();
```

Comment:
This removal gave me some headaches... I am using a TinyMCE editor inside the `fileEditFields` of an `UploadField` and after this commit I need to load that JS by myself (see ntd/silverstripe-carousel@a729b710cb3d8cad7f0843cb84e07f1c27728245).

Not sure it is a regression or something expected though.